src/cryptoadvance/specter/rpc.py

```diff
@@ -18,7 +18,12 @@ def get_default_datadir():
 def get_rpcconfig():
     path = get_default_datadir()
     config = {
-        "bitcoin.conf": {},
+        "bitcoin.conf": {
+            "default": {},
+            "mainnet": {},
+            "testnet": {},
```

Comment:
please use `main` and `test` as keys here, otherwise lookups in RPC_PORTS don't work and other things are more complicated as well. See comments below.